app/styles/bootstrap.less

```diff
@@ -0,0 +1,49 @@
+// Core variables and mixins
+@import "../../node_modules/bootstrap/less/mixins";
+
+// Reset and dependencies
+@import "../../node_modules/bootstrap/less/normalize";
+@import "../../node_modules/bootstrap/less/print";
+@import "../../node_modules/bootstrap/less/glyphicons";
+
+// Core CSS
+@import "../../node_modules/bootstrap/less/scaffolding";
+@import "../../node_modules/bootstrap/less/type";
+@import "../../node_modules/bootstrap/less/code";
+@import "../../node_modules/bootstrap/less/grid";
+@import "../../node_modules/bootstrap/less/tables";
+@import "../../node_modules/bootstrap/less/forms";
+@import "../../node_modules/bootstrap/less/buttons";
+
+// Components
+@import "../../node_modules/bootstrap/less/component-animations";
+@import "../../node_modules/bootstrap/less/dropdowns";
+@import "../../node_modules/bootstrap/less/button-groups";
+@import "../../node_modules/bootstrap/less/input-groups";
+@import "../../node_modules/bootstrap/less/navs";
+@import "../../node_modules/bootstrap/less/navbar";
+@import "../../node_modules/bootstrap/less/breadcrumbs";
+@import "../../node_modules/bootstrap/less/pagination";
+@import "../../node_modules/bootstrap/less/pager";
+@import "../../node_modules/bootstrap/less/labels";
+@import "../../node_modules/bootstrap/less/badges";
+@import "../../node_modules/bootstrap/less/jumbotron";
```

Comment:
This is a 1:1 copy of the requires from bootstrap itself, I have not removed any unused code yet.